frontend/package.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
package-lock.json shall not be added into the repo. Please change the gitignore file

Reply:
added